main-site/lib/data.ts

```diff
@@ -17,3 +17,82 @@ export const aboutSectionData: AboutSectionData[] = [
       'Expand beyond your horizons and grow your current skill set in a safe and supportive environment.'
   }
 ];
+
+export const FaqSectionData: FaqPropsArray[] = [
```

Comment:
Can we rename this to FaqSectionData? Props represents what we're going to pass into a component